### DISCUSSION TITLE
bugs on Billable/Non-billable from the modifications

### DISCUSSION THREAD

**Discussion by SBPJ** [kimai/kimai#3941] [General]
Version: 1.30.9 stable 

When users select a Billable activity, it is recorded as Billable initially.
However, even if user modifies to Non-billable activity, that activity remains Billable.

Vice versa, when user creates a non-billable first, then later modifies it to a Billable activity  it still remains non-billable although the activity is billable.

Users don't have the right to select billable status, but they can change anything with the above method.

**Comment by kevinpapst** [comment]
Yes. Same as hourly rate. That is the way Kimai 1.30 works.

This behavior is changed for 2.0, which will be released soon.
In the new version all relevant fields are reset upon change of project/user/activity.

**Reply by SBPJ** [reply]
Thanks for your comment.   Hope 2.0 solve this issue.

**Reply by kevinpapst** [reply]
Try the demo and help testing!
